src/helpers/getDXContractAddresses.js

```diff
@@ -0,0 +1,35 @@
+const Network2Id = {
```

Comment:
I like this util, but I think we can get the network id from web3 instead of this map